content/en/docs/concepts/architecture/nodes.md

```diff
@@ -14,7 +14,12 @@ A node may be a virtual or physical machine, depending on the cluster. Each node
 is managed by the
 {{< glossary_tooltip text="control plane" term_id="control-plane" >}}
 and contains the services necessary to run
-{{< glossary_tooltip text="Pods" term_id="pod" >}}.
+{{< glossary_tooltip text="Pods" term_id="pod" >}}. A node object is an object in 
```

Comment:
We use the resource name (i.e. `Node`) when talking about resource objects or resource types.

```suggestion
{{< glossary_tooltip text="Pods" term_id="pod" >}}. A Node object is an object in 
```